vm/interpreter/src/vm.rs

```diff
@@ -20,9 +20,10 @@ use forest_encoding::Cbor;
 use ipld_blockstore::BlockStore;
 use log::debug;
 use message::{ChainMessage, Message, MessageReceipt, UnsignedMessage};
-use networks::{UPGRADE_CLAUS_HEIGHT, UPGRADE_PLACEHOLDER_HEIGHT};
+use networks::{UPGRADE_ACTORS_V4_HEIGHT, UPGRADE_CLAUS_HEIGHT};
```

Comment:
Not directly related to this PR, but is this spelled correctly?

Reply:
Um, not sure

Reply:
Which word? I don't see any misspellings...

Reply:
`UPGRADE_CLAUS_HEIGHT` ?

Reply:
what is CLAUS?

Reply:
Santa Claus lmao. It was an emergency upgrade that happened a few days before christmas last year

Reply:
oh
I thought, is this supposed to be CLASS? CLAUSE? was not expecting the name of a deity